src/openforms/js/components/admin/form_design/FormConfigurationFields.js

```diff
@@ -308,6 +309,25 @@ const FormConfigurationFields = ({
           />
         </Field>
       </FormRow>
+      <FormRow>
+        <Checkbox
+          name="form.suspendAllowed"
```

Comment:
please add the default value of `True` to `form-creation-form.js`, in the `initialFormState` constant.